src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -724,6 +718,14 @@ NOINLINE freeze_result FreezeBase::freeze_slow() {
   log_develop_trace(continuations)("freeze_slow  #" INTPTR_FORMAT, _cont.hash());
   assert(_thread->thread_state() == _thread_in_vm || _thread->thread_state() == _thread_blocked, "");
 
+#if CONT_JFR
+  EventContinuationFreezeSlow e;
+  if (e.should_commit()) {
```

Comment:
Committing the event should happen at the end of the method to measure the time.

Reply:
That makes sense but the time recorded for these is startTime:
jdk.ContinuationThawSlow {
  startTime = 11:54:23.301 (2022-09-14)
  id = 4260565248
  eventThread = "MainThread" (javaThreadId = 36)
}